fast64_internal/panels.py

```diff
@@ -58,3 +58,11 @@ class MK64_Panel(bpy.types.Panel):
     @classmethod
     def poll(cls, context):
         return context.scene.gameEditorMode == "MK64"
+
+    @classmethod
+    def game_check(context):
+        """
+        Provides shorthand method to check context for
+        panels with more complex conditions
+        """
+        return context.scene.gameEditorMode == "MK64"
```

Comment:
```suggestion
```

Reply:
I wrote this quickly and it was not checked to be fitting with the rest of the repo, megamech took my prototype and finished it
So the stuff I made was not all to PR standards, evaluate it as if megamech made it

Reply:
@MegaMech Could we go back to the initial folder structure if you have the time to make those changes? I'll review functionality some other time I have to setup mk64 again